Wikipedia/Code/GroupedAccessibilityView.swift

```diff
@@ -0,0 +1,71 @@
+import UIKit
+
+class GroupedAccessibilityView: UIView {
+
+    var arrangedAccessibilityViews: [UIView] = []
+    
+    //convienene outlets
+    @IBOutlet weak var accessibilityView0: UIView?
```

Comment:
Can't really guarantee the order of an outlet collection - we could tag all the views and sort on that but this seems easier? Open to suggestion